recipe/meta.yaml

```
@@ -21,13 +21,13 @@ requirements:
     - {{ compiler('c') }}
   host:
     - python
-    - cryptography
-    - cython
+    - cryptography 39.0.1
```

```suggestion
    - cryptography 38.0.4
```
(as we don't have 39.0.1 built on 3.11 yet)

---

👍 Good catch. Thank you!